src/Workspaces/Core/Portable/Shared/Extensions/ITypeSymbolExtensions.cs

```diff
@@ -210,6 +212,13 @@ public static ITypeSymbol RemoveNullableIfPresent(this ITypeSymbol symbol)
             }
 
             return arrBuilder.ToImmutableAndFree();
+
+            // local functions
+
+            static Task<Compilation> GetCompilationOrNull(Project project, CancellationToken cancellationToken)
+                => project == null ?
+                    Task.FromResult<Compilation>(null) :
+                    project.GetCompilationAsync(cancellationToken);
```

Comment:
1.  `?:` should wrap such that `?` and `:` are at the start of the line.
2. this can just be `project?.GetCompilationAsync(...) ?? Task.FromResult<Compilation>(null);`

Reply:
Also, getting a null project simply seems like a bug.  What causes that here?  I feel like this papers over a bug earlier on.

Reply:
Tests, or anything that passes null document with SymbolAndProjectId